README.md

```diff
@@ -158,6 +158,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
   * [Noddity](http://noddity.com/) - It's a blog, it's a wiki, it's a fast CMS. ([Source Code](https://github.com/TehShrike/noddity)) `WTFPL` `Nodejs`
   * [PluXml](http://www.pluxml.org/) - XML-based blog/CMS platform. ([Source Code](https://github.com/pluxml/PluXml)) `GPL` `PHP`
   * [Postleaf](https://www.postleaf.org/) - An open source blogging platform with inline editing, handlebar templates, and a beautiful user interface. ([Source Code](https://github.com/Postleaf/postleaf)) `GPLv3` `PHP`
+  * [Solo](https://github.com/b3log/solo) - A blogging system written in Java, feel free to create your or your team own blog. ([Demo](http://88250.b3log.org/), [Source Code](https://github.com/b3log/solo)) `Apache` `Java`
```

Comment:
Please remove the Source Code link as it is the same as the homepage URL and duplicate links are not allowed.

      * [Solo](https://github.com/b3log/solo) - A blogging system written in Java, feel free to create your or your team own blog. ([Demo](http://88250.b3log.org/)) `Apache` `Java`